### ISSUE TITLE
Cannot find site build with octopress build/octopress deploy commands

### ISSUE THREAD

**Issue by bg-scro** [octopress/octopress#151]
Have been trying to get Octopress working on Github pages for several days and having limited success. Can't seem to find good documentation on Octopress 3 anywhere so I've been working through a mix of steps from various blogs but nothing seems to work. Went back to a clean setup and tried to go through the steps on your github page readme. I've generated some dummy pages and posts with `octopress new page` and ` octopress new post`. These appear to be working fine.

I ran `octopress deploy init git <SITE URL>` successfully. However, running `octopress deploy` generates the error:

`Cannot find site build at /Users/me/sites/octo-4/_site. Be sure to build your site first.`

However, `octopress build` does nothing.

This is really frustrating and doesn't seem to be documented anywhere. Please help!

By the way, my Octopress folder tree:
.
├── _deploy.yml
├── _drafts
├── _posts
│   ├── 2015-11-09-testing-a-draft.markdown
│   └── 2015-11-09-testing-octopress.markdown
├── _templates
│   ├── draft
│   ├── page
│   └── post
└── about.md

**Comment by plttn** [comment]
In short, Octopress doesn't shim commands for Jekyll anymore. Building from
Jekyll should cause it to work.

**Comment by bg-scro** [comment]
Thanks for the quick response. So I ran `jekyll build` and that seemed to work - it generated the _site folder. I then ran `octopress deploy`, which pushed to Github.

I think what is very confusing is that it's not entirely clear in the documentation that you first need a jekyll site, then run `octopress init` within that project folder to create the _templates folder, and proceed with the other octopress commands.
